packages/web-components/src/components/va-official-gov-banner/va-official-gov-banner.tsx

```diff
@@ -0,0 +1,209 @@
+      el.innerHTML = text.replaceAll(`.${this.tld}`, `<strong>.${this.tld}</strong>`);
+    }
+  }
+
+  /**
+   * This will add <strong> tags around various words in the text that is provided
+   * as well as add the SVG lock image.
+   */
+  private govSiteLockText = () => {
+    const el = this.el.shadowRoot?.querySelector('.gov-site-lock-text') as HTMLElement;
+
+    // eslint-disable-next-line i18next/no-literal-string
+    const lockSvg = `(&nbsp;<span class="icon-lock"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" role="img" aria-labelledby="banner-lock-description" focusable="false">
+    <title id="banner-lock-title">Lock</title>
+    <desc id="banner-lock-description">Locked padlock icon</desc>
+    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
+    </svg></span>&nbsp;)`
+  
+    let html = i18next.t('gov-site-lock', { image: 'SVG', tld: this.tld });
+
+    html = html.replace('SVG', lockSvg);
+    html = html.replace('lock', `<strong>lock</strong>`);
+    html = html.replace('candado', `<strong>candado</strong>`);
+    html = html.replace('https://', `<strong>https://</strong>`);
+
+    if (el) {
+      el.innerHTML = html;
+    }
+  }
+
+  render() {
+    const { tld } = this;
+    if (tld === 'gov' || tld === 'mil') {
+      return (
+        <Host>
+          <section
+            class="banner"
+            aria-label={i18next.t('gov-site-label')}
+          >
+            <div class="accordion"  >
+              <header >
+                <div class="inner">
+                  <div class="grid-col-auto">
+                    <img 
+                      role="presentation" 
+                      class="header-flag" 
+                      src="https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/tiny-usa-flag.png"
+                      alt="" />
+                  </div>
+                  <div class="grid-col-fill">
+                    <p class="header-text">{i18next.t('gov-site-label')}</p>
+                    <p class="header-action" aria-hidden="true">{i18next.t('gov-site-button')}</p>
+                  </div>
+                  <button 
+                    onClick={this.handleClick}
+                    type="button" 
+                    aria-expanded="false" 
+                    aria-controls="official-gov-banner">
+                    <span class="button-text">{i18next.t('gov-site-button')}</span>
+                  </button>
+                </div>
+              </header>
+  
+              <div class="content" id="official-gov-banner" hidden>
+                <div class="grid-row">
+                  <div class="col">
+                    <img 
+                      src="https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/icon-dot-gov.svg" 
```

Comment:
This was taken directly from the existing VA.gov implementation. I sometimes see a slight delay with this is loaded because there's a call out to AWS for it so if we're not happy with that behavior, I can see if there is a way to load without hot linking.

![Screenshot 2023-02-01 at 3 17 13 PM](https://user-images.githubusercontent.com/872479/216165808-6ccd1280-5e89-45d7-916c-449567646889.png)

Reply:
I attempted to get this `png` image to load locally using the [Stencil asset base path](https://stenciljs.com/docs/v2/assets#assets) but wasn't successful. This might be a [limitation of Stencil](https://github.com/ionic-team/stencil/issues/1868). For now we will probably need to keep the AWS link. We are able to import SVGs though.